### ISSUE TITLE
Data should be saved in a database rather than in local json files

### ISSUE THREAD

**Issue by martinlevesque** [lovasoa/whitebophir#2]
The boards data are currently saved as json files as far as I can see. They should be saved using any database.

**Comment by dazsmith** [comment]
Json has the advantage of very easy automated backup, server side.

**Comment by lovasoa** [comment]
yes, saving boards as simple files also has its advantages. Ideally, we would have several persistence backends between which the user could choose. 

see https://github.com/lovasoa/whitebophir/pull/21
